submissions/JuliaVorotchenko/Popup-task/main.css

```diff
@@ -0,0 +1,368 @@
+    margin-bottom: 5px;
+    justify-content: space-around;
+}
+
+.app-list-item {
+    width: 90px;
+    height: 90px;
+    padding: 2px;
+    display: flex;
+    justify-content: center;
+}
+
+.app-link {
+    width: 100%;
+    height: 100%;
+    border: 1px solid transparent;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family: 'Work Sans', sans-serif;
+    color: #4a5b56;
+    text-decoration: none;
+    font-size: 400;
+}
+
+.app-list-item:first-child .app-icon {
+    background: url(img/gmail.svg);
+    background-size: cover;
+}
+
+.app-list-item:nth-child(2) .app-icon {
+    background: url(img/youtube.svg);
+    background-size: contain;
+}
+
+.app-list-item:nth-child(3) .app-icon {
+    background: url(img/hangouts.svg);
+    background-size: contain;
+}
+
+.app-list-item:nth-child(4) .app-icon {
+    background: url(img/photos.svg);
+    background-size: contain;
+}
+
+.app-list-item:nth-child(5) .app-icon {
+    background: url(img/calendar.svg);
+    background-size: contain;
+}
+
+.app-list-item:nth-child(6) .app-icon {
+    background: url(img/translate.svg);
+    background-size: contain;
+}
+
+.app-list-item:nth-child(7) .app-icon{
+    background: url(img/weather.svg);
+    background-size: contain;
+}
+
+.app-list-item:nth-child(8) .app-icon {
+    background: url(img/note.svg);
+    background-size: contain;
+}
+
+.app-list-item:last-child .app-icon{
+    background: url(img/account.svg);
+    background-size: contain;
+}
+
+.app-icon {
+    height: 55px;
+    width: 55px;
+    display: block;
+}
+
+.app-link-name {
+    font-size: 16px;
+    font-family: 'Work Sans', sans-serif;
+    display: block;
+}
+
+.app-hidden-list-item:first-child .app-icon {
+    background: url(img/phone-book.svg);
+    background-size: contain;
+}
+
+.app-hidden-list-item:nth-child(2) .app-icon {
+    background: url(img/maps-and-location.svg);
+    background-size: contain;
+}
+
+.app-hidden-list-item:last-child .app-icon {
+    background: url(img/drive.svg);
+    background-size: contain;
+}
+
+.popup-button {   
+    display: block;
+    width: 100%;
+    background: #f5f5f5;
+    cursor: pointer;
+    padding: 5px;
+    width: 100%;
+    box-sizing: border-box;
+    text-align: center;
+    font-size: 14px;
+    border: 1px solid transparent;
+}
+
+.division-line {
+    margin: 0 auto;
+    background: #ebebeb;
+    width: 100%;
+    height: 1px;
+    
+}
+
+.popup-wrapper:after,
+.popup-wrapper:before {
+  content: "";
+  display: block;
+  position: absolute;
+  z-index: 1;
+  width: 0;
+  height: 0;
+}
+
+.popup-wrapper:after {
+  border-color: transparent;
+  border-bottom: solid #fff;
+  border-style: dashed dashed solid;
+  border-width: 0 8.5px 8.5px;
+  right: 134px;
+  top: -9px;
+}
+
+.popup-wrapper:before {
+  border-color: transparent;
+  border-style: dashed dashed solid;
+  border-width: 0 8.5px 8.5px;
+  right: 134px;
+  border-bottom-color: rgba(0, 0, 0, 0.2);
+  top: -10px;
+}
+
```

Comment:
`.popup-wrapper:after` {
    border-bottom: solid #edf9f6;
    top: -8px;
}`
`.popup-wrapper:before {
    border-bottom-color: rgb(210, 213, 212);
    top: -9px;
}`

It looks like better))
(I change only attribute color and top)